std/async/delay_test.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import { delay } from "./delay.ts";
+import { assert } from "../testing/asserts.ts";
+
+Deno.test("[async] delay", async function (): Promise<void> {
+  const start = new Date();
+  const delayedPromise = delay(100);
+  const result = await delayedPromise;
+  const diff = new Date().getTime() - start.getTime();
+  assert(result === undefined);
+  assert(diff >= 100);
+  assert(diff < 150);
```

Comment:
Thanks for the patch! The test looks good except this last assert - this is guaranteed by the API - please remove.

Reply:
Thanks for the fast review - just pushed the change you've requested.